buildenv/jenkins/openjdk_tests

```diff
@@ -499,6 +499,7 @@ def runTest() {
                     errorList.add(".*java.io.IOException: Cannot run program \"nohup\".*");
                     errorList.add(".*unzip: command not found.*");
                     errorList.add(".*ant: command not found.*");
+                    errorList.add(".*ERROR: Cannot delete workspace :Unable to delete .*");
```

Comment:
Is that 2nd colon in the right place?  Not sure if we're matching exact strings.

Reply:
The message is correct. However, I will update it to accept space in case this gets fixed by Jenkins.